packages/mds-db/events.ts

```diff
@@ -280,15 +280,15 @@ export async function readEventsWithTelemetry({
     }
   }
 
+  // we can only select based on event criteria
   const where = conditions.length ? ` WHERE ${conditions.join(' AND ')}` : ''
 
   const { rows } = await exec(
-    `SELECT E.*, T.lat, T.lng, T.speed, T.heading, T.accuracy, T.altitude, T.charge, T.timestamp AS telemetry_timestamp FROM (SELECT * FROM ${
-      schema.TABLE.events
-    }${where} ORDER BY ${order_by} LIMIT ${vals.add(limit)}
-    ) AS E LEFT JOIN ${
-      schema.TABLE.telemetry
-    } T ON E.device_id = T.device_id AND CASE WHEN E.telemetry_timestamp IS NULL THEN E.timestamp ELSE E.telemetry_timestamp END = T.timestamp ORDER BY ${order_by}`,
+    `SELECT E.*, T.lat, T.lng, T.speed, T.heading, T.accuracy, T.altitude, T.charge, T.timestamp AS telemetry_timestamp 
```

Comment:
Is there any change here besides formatting? Hard to grok such a long SQL statement

Reply:
Kym spotted an erroneous extra `ORDER BY`.  Otherwise, just formatting.

Reply:
I think the prior SQL was correct; see comment below.